app/views/companies/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= form_for(@company) do |f| %>
+  <%= f.label :name   %>
+  <%= f.text_field :name %>
+
+  <%= f.label :street_address %>
+  <%= f.text_field :street_address %>
+
+  <%= f.label :city_state_zip %>
+  <%= f.text_field :city_state_zip %>
+
+  <%= f.label :city %>
+  <%= f.collection_select :city_id, City.all, :id,:name %>
+
+  <%= f.label :category %>
+  <%= f.collection_select :category_id, Category.all, :id,:name %>
+
+  <%= f.label :state %>
+  <%= f.collection_select :state_id, State.all, :id,:name %>
+
+  <%= f.label :zip_code %>
+  <%= f.collection_select :zip_code_id, ZipCode.all, :id,:zip_code %>
```

Comment:
I like how we're splitting out city, state and zipcode into their own tables.  We must remember to seed that or else none of this will work.